### DISCUSSION TITLE
[Feature] Photos without Album

### DISCUSSION THREAD

**Discussion by MickLesk** [immich-app/immich#6622] [Feature Request]
### The feature

This is an issue that really annoys me with Google Photos.
I would like to have an overview of all photos that do not belong to an album.
I would also like to be able to simply use a "search string" to add them to an album or simply "delete" them. 
But also via a built-in filter in the header or as a single menu item (dropdown?)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by dmeireles** [comment]
Hey. Had this same question/request, and to my surprise, Immich already has this. Just use the search bar filters, there's an option there to only list results for items that don't belong to any album

**Comment by CrazyWolf13** [comment]
I really like this idea and would love to see this implemented in immich, I think when doing a lot of importing an reorganizing, this would be areally useful feature, to ensure each photo is assigned to an album

**Comment by CrazyWolf13** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/2559 which is a duplicate of https://github.com/immich-app/immich/discussions/1621

Solution: https://github.com/immich-app/immich/discussions/1621#discussioncomment-8570436
